### DISCUSSION TITLE
Question about speed

### DISCUSSION THREAD

**Discussion by BinaryQuantumSoul** [rupeshs/fastsdcpu#86] [General]
Hello, I tried LCM-OpenVino generations on my Intel i5 8thGen, 
generation time for a 1-step 512x512 image is around 100 seconds
 (sometimes it is killed with `./start-webui.sh: línea 25:  9319 Terminado (killed)      $PYTHON_COMMAND src/app.py -w`).
 Is that normal speed for fastsdcpu ?

**Comment by rupeshs** [comment]
Yes,First time it will take some time, downloading models etc

**Reply by BinaryQuantumSoul** [reply]
It is the time after downloading the models

**Reply by rupeshs** [reply]
For openvino model compilation will take some time but faster inference. Default pytorch mode model loading is comparatively faster but inference speed will be slower compared to OpenVINO.
Also refer this : https://github.com/rupeshs/fastsdcpu#fast-1-step-inference-sdxl-turbo---adversarial-diffusion-distillationadd
